src/Pages/Main/Homeitem/Homeitem.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import HomeCard from "./HomeCard";
+import "./Homeitem.scss";
+import HOMECARD from "./HomeData";
+
+class Homeitem extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      CardList: HOMECARD,
+    };
+  }
```

Comment:
```suggestion
  state = {
    cardList: HOMECARD,
  }
```
- shortcut으로 이렇게 간단히 쓸 수도 있습니다.
- 변수명은 카멜 케이스로 작성해주세요!